static/src/javascripts/es6/projects/common/modules/crosswords/clues.js

```diff
@@ -1,9 +1,10 @@
 /* jshint newcap: false */
+import classNames from 'classnames';
+import React from 'react';
+
 import _ from 'common/utils/_';
-import React from 'react/addons';
 
-var classSet = React.addons.classSet,
-    Clue = React.createClass({
+var Clue = React.createClass({
```

Comment:
That's a shame about mixins, though I guess if that's gonna be the case with all ES6 classes, we maybe just ought to be used to it by now, and not expect it of React. (As that means you have to context switch when using React classes, instead of just treating them like any other JavaScript classes.)